src/components/header-block/header-block.tsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import styled from 'styled-components';
+import { css } from 'styled-components';
+import { Illustration, IllustrationSize } from '../illustration/illustration';
+import { IllustrationType } from '../illustration/illustration-set';
+import { down, up } from '../breakpoint/breakpoint';
```

Comment:
2 imports are not used and can be removed:

<img width="542" alt="Screenshot 2021-11-23 at 11 17 11" src="https://user-images.githubusercontent.com/7814926/143006687-63ea7753-9e25-422b-913d-798f927305c6.png">